src/plugins/plugin.decimation.js

```diff
@@ -196,7 +247,10 @@ export default {
         return;
       }
 
-      if (data.length <= 4 * availableWidth) {
+      // We know that the diagram is a line type
```

Comment:
I don't really get the intention of this comment? If its to state that we have passed the above if's, then I'd remove it.

Reply:
Residual code from my tests